BasePolicies/CODE_OF_CONDUCT.md

```diff
@@ -1,27 +1,25 @@
 ## Working Group Code of Conduct Template
 
 Every Working Group must have a Code of Conduct. By default,
-all repositories under the ASC Software Organization GitHub Organization fall under
-the [ASC Software Organization Code of Conduct][], but Working Groups can choose to
+all repositories under the Planetary Software Organization fall under
+the [Planetary Software TSC Code of Conduct](https://github.com/planetarysoftware/TSC/blob/master/CODE_OF_CONDUCT.md). Working Groups can choose to
```

Comment:
No need to modify this link. If you update the link in the bottom of the doc, all of the corresponding links will update too.